toc/working-groups/foundational-infrastructure.md

```diff
@@ -171,9 +171,9 @@ areas:
     github: coolgang123
   - name: Daniel Garnier-Moiroux
     github: kehrlann
-  reviewers:
   - name: Filip Hanik
     github: fhanik
+  reviewers:
```

Comment:
```suggestion
```

Reply:
think you can remove it

Reply:
yes, it should be removed or an empty array is here: https://github.com/cloudfoundry/community/blob/a338b5850bd300911c818d4883bfbd81e33e1aa4/toc/working-groups/paketo.md?plain=1#L72